cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -95,7 +102,9 @@ function renderGenericComponent() {
                      node.mount(container);
                      componentArr[paymentMethod.type] = node;
                  }
-                 catch (e) {}
+                 catch (e) {
+                    console.error(e);
```

Comment:
Do we have to log this when there is an error? Maybe we can just skip it and proceed with next component instead.